modules/local/drop_sample_annot.nf

```diff
@@ -8,23 +8,33 @@ process DROP_ANNOTATION {
         'biocontainers/drop:1.3.3--pyhdfd78af_0' }"
 
     input:
+    path(bam)
+    val(samples)
     path(processed_gene_counts)
+    path(ref_annot)
     path(gtf)
 
     output:
-    path('sample_annotation.tsv'), emit: sample_annotation_drop
-    path "versions.yml"              , emit: versions
+    path('sample_annotation.tsv'), emit: drop_annot
+    path "versions.yml"   , emit: versions
 
     when:
     task.ext.when == null || task.ext.when
 
     script:
-    def gtf_name   = gtf ? gtf.getBaseName() : ""
+    def strandedness = samples ? "--strandedness ${samples.strandedness}" : ""
+    def input_samples = samples ? "${samples.id}" : ""
```

Comment:
No, samples will always be given, samples is the name of the samples analysed, it is a channel generated from meta as star_samples

Reply:
```suggestion
    def strandedness = samples ? "--strandedness ${samples.strandedness}" : ""
```
Maybe we should remove this part then as you here check if samples is given and otherwise sets the string to empty. See comment below . Same for strandedness if it is a mandatory parameter